### DISCUSSION TITLE
Proposal to increase Governor limits for Sei Blockchain

### DISCUSSION THREAD

**Discussion by alvinkan** [wormhole-foundation/wormhole#3303] [Governance] [answered]
I would like to open up a discussion for Governor parameters for Sei. We have seen significant inbound/outbound flows on Wormhole for Sei since it's launch.  Earlier this week Wormhole increased the Governor limits from $200K/$20K, then $500K/50K, and then $1M/$100K, but we consistently see flows that max out the Governor capacity causing delays for users up to 24hrs.

I'd like to propose increasing the existing limit from $1M/$100K to $5M/$500K to offer some headroom to allow for additional organic growth.

**Comment by tonyjin** [comment]
Chiming in - data from one of the Wormhole guardians 19 hours ago showed 1717 transactions from Sei held by the Governor for a total of $7,083,474 in notional value, which supports the argument of increasing limits to $5M/$500K.

There appears to be a large active Sei airdrop campaign (https://blog.sei.io/the-sei-airdrop/) that incentivizes bridging assets onto Sei from a huge number of allowlisted wallets (1.5M addresses per https://twitter.com/Sei_FND/status/1692070808783368592). It seems likely that Governor limits will continue to be hit regardless of what they are raised to as long as this campaign is active.

While the data so far supports increasing limits to $5M/$500K as Wormhole has done for other major launches (e.g. https://github.com/wormhole-foundation/wormhole/discussions/2890), I'd caution against continued increases until the chain is more mature.

The goal of the Governor (https://github.com/wormhole-foundation/wormhole/blob/main/whitepapers/0007_governor.md) is to protect Wormhole-connected chains against cross-chain contagion risk, which is higher in the early days of a new chain.

**Reply by evan-gray** [reply]
Update: there are now ~800 transfers held totaling ~$4m. For anyone interested, these are visible by viewing individual guardian's stats on the [dashboard](https://wormhole-foundation.github.io/wormhole-dashboard/) (drop-down on the top right).

**Reply by evan-gray** [reply]
Update again: ~446 for ~$1.6m. Would be interested to see how much volume holds up given the decrease in inflows. Here's the In/Out by day for VAAs (does not include those held, so ~$1.6 on the outbound side within the last 24 hours)

### In

Row | tx_date | count | notional_usd 
-- | -- | -- | -- 
1 | 2023-08-20 | 593 | 1,420,185 
2 | 2023-08-19 | 1036 | 2,869,265 
3 | 2023-08-18 | 1525 | 5,434,585 
4 | 2023-08-17 | 6032 | 13,145,725 
5 | 2023-08-16 | 4061 | 2,312,495 
6 | 2023-08-15 | 10122 | 134,126 

### Out

Row | tx_date | count | notional_usd 
-- | -- | -- | -- 
1 | 2023-08-20 | 151 | 22,612 
2 | 2023-08-19 | 1115 | 3,853,184 
3 | 2023-08-18 | 2221 | 6,555,584 
4 | 2023-08-17 | 4397 | 7,966,668 
5 | 2023-08-16 | 1200 | 174,607 
6 | 2023-08-15 | 64 | 10,198

**Reply by claudijd** [reply, marked as answer]
Here's PR to reflect the proposal of a $5M/$500K limit increase from $1M/$100K limit on Sei.  This should be more than sufficient to consume the backlog ($1.5M) and offer some room for organic growth.

https://github.com/wormhole-foundation/wormhole/pull/3306

**Comment by tbjump** [comment]
Since Sei not only has large outflows but also large inflows, I think that just increasing the governor limit on Sei will lead to exhaustions on other chains. I'm mainly thinking about Arbitrum and Optimism here, which should be considered to be 5x'd as well.
